style.css

```diff
@@ -0,0 +1,54 @@
+#something {
+  text-align: center;
+}
+
+#somethinmore {
+  align-items: center;
+  padding-bottom: 2em;
+  padding-top: 2em;
+}
+
+#form {
+  text-align: center;
+}
+body {
+  background-color: rgb(230, 217, 201);
+}
+#form {
+  padding-top: 7em;
+  padding-bottom: 2em;
+  align-items: center;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  display: flex;
+  top: 0;
+  left: 0;
+  min-height: 30px;
+  justify-content: space-around;
+  background-color: rgb(230, 217, 201);
+  padding: 8px 2px;
+}
+
+@media (max-width: 600px) {
+  header {
+    flex-wrap: wrap;
+  }
+}
+
+#evenmore {
+  border: 3px;
+  border-radius: 2px;
+  text-align: center;
+}
+
```

Comment:
Please use Flexbox to align this section as is required to approve this project and makes it look nicer.

![prices](https://user-images.githubusercontent.com/111214189/192381176-f13b058f-ead9-4eec-9914-29b8461c3c80.png)

Reply:
- [ ] Good job using the inline comments and supporting the comment with a screenshot of the bug :100: . However, I would highly recommend also including links that can provide further information to the student, for example, in this case you could add a link to the project requirements :bulb:

Reply:
> Please use Flexbox to align this section as is required to approve this project and makes it look nicer.
> 
> ![prices](https://user-images.githubusercontent.com/111214189/192381176-f13b058f-ead9-4eec-9914-29b8461c3c80.png)

You can find the requeriments for this project here 👍  : https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-projects/build-a-product-landing-page